### PR TITLE
Modify rule S6709: Add how to fix it for Scikit-learn

### PR DESCRIPTION
**Pull request by joke1196**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S6709/python/how-to-fix-it/sklearn.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+=== Code examples
+
+==== Noncompliant code example
+
+[source,python,diff-id=1,diff-type=noncompliant]
+----
+from sklearn.model_selection import train_test_split
+from sklearn.datasets import load_iris 
+
+X, y = load_iris(return_X_y=true)
```
Comment (ghislainpiot) [inline rules/S6709/python/how-to-fix-it/sklearn.adoc:14]: true -> True

**Review by ghislainpiot — APPROVED** [review]
Looks good to me! Nice catch on separating the 2 libraries in different files.

File: rules/S6709/python/how-to-fix-it/sklearn.adoc (hunk 1 above)
Comment (joke1196) [reply rules/S6709/python/how-to-fix-it/sklearn.adoc:14]: Ohh nice catch thanks!

**Review by jean-jimbo-sonarsource — APPROVED** [review]
